packet/adaptationfield/create.go

```diff
@@ -4,13 +4,13 @@ import "github.com/Comcast/gots/packet"
 
 func SetPrivateData(pkt *packet.Packet, af []byte) {
```

Comment:
Note that the potential for failure here suggests that `SetPrivateData` should return an error.